File: catkin_ws/src/liveDemo/src/mypub.cpp
#include "ros/ros.h"
#include "liveDemo/my_msg.h"
#include "mylib.h"
#include "geometry_msgs/Twist.h"

int main(int argc, char **argv)
{
	ros::init(argc,argv,"mypub_node");
	
	ros::NodeHandle my_handle;

	ros::Publisher mypub_object = my_handle.advertise<geometry_msgs::Twist>("cmd_vel", 100);

	ros::Rate loop_rate(10);

	int counter = 0;

	while(ros::ok())
	{
		loop_rate.sleep();
		
		geometry_msgs::Twist mypub_msg; // replace existing similar assignments
		                mypub_msg.linear.x = 1;
						mypub_msg.linear.z = 0.1;
		                mypub_msg.angular.z = 0.5;

		mypub_object.publish(mypub_msg);
	}

	return 0;
}

File: catkin_ws/src/py_demo/src/mypub.py
#!/usr/bin/env python

import rospy
from geometry_msgs.msg import Twist 

def mypub_node():
    mypub_object = rospy.Publisher('cmd_vel', Twist, queue_size=100) #my_handle.advertise<Twist>("cmd_vel", 100);

    rospy.init_node('mypub_node')
    rate = rospy.Rate(10) # 10hz

    counter = 0

    while not rospy.is_shutdown():
        rate.sleep()

        mypub_msg = Twist() # replace existing similar assignments
        mypub_msg.linear.x = 1
        mypub_msg.linear.z = 0.1
        mypub_msg.angular.z = 0.5

        mypub_object.publish(mypub_msg)

if __name__ == '__main__':
    try:
        mypub_node()
    except rospy.ROSInterruptException:
        pass